target/linux/rockchip/Makefile

```diff
@@ -17,7 +17,7 @@ endef
 
 include $(INCLUDE_DIR)/target.mk
 
-DEFAULT_PACKAGES += uboot-envtools partx-utils e2fsprogs mkf2fs
+DEFAULT_PACKAGES += uboot-envtools partx-utils e2fsprogs mkf2fs kmod-phy-realtek
```

Comment:
Doesn't this get pulled in since a bunch of other ```CONFIG_ROCKCHIP_*```are enabled in the kernel config?

Reply:
without it I see not  `PHY [stmmac-0:00] driver [RTL8211F Gigabit Ethernet]`

Reply:
seems to work ok with `rk_gmac-dwmac ff540000.ethernet eth0: PHY [stmmac-0:00] driver [Generic PHY]`